pkg/controller/infrastructure/actuator.go

```diff
@@ -54,7 +54,6 @@ func newTerraformer(logger logr.Logger, restConfig *rest.Config, purpose string,
 	owner := metav1.NewControllerRef(infra, extensionsv1alpha1.SchemeGroupVersion.WithKind(extensionsv1alpha1.InfrastructureResource))
 	return tf.
 		UseV2(true).
```

Comment:
Side note out of context of this PR: we can also consider making v2 the default mode in the terraform pkg.